### DISCUSSION TITLE
Just installed, everything is 'NO DATA'

### DISCUSSION THREAD

**Discussion by gurmukhp** [teslamate-org/teslamate#3274] [Getting started]
Hey there

I've just setup TeslaMate and wasn't able to view the tables in the database because the password was incorrect. I fixed it and can confirm I'm able to view each table (there are no errors).

I can see the current charge of the car and can see that its parked.

However all dashboards so no data at all, is this expected? Even the location on the TeslaMate's homepage (:3000) is just a blue map, the status of the car (asleep for 29 mins), is correct.

Is everything setup and I just need to drive it for the tables to populate or could there be something wrong?

**Comment by Astyan26** [comment]
Hello, i have the same problem... I try to resolv it too...

**Comment by Astyan26** [comment]
hi , il find this and it solve my problem :)

https://github.com/adriankumpf/teslamate/discussions/2773#discussioncomment-3191930
I hope you too

**Comment by gurmukhp** [comment]
I was able to solve this issue by turning the car on and then driving for 1m forwards, then backwards
